Update installing.rst

nworbnhoj opened the pull request:
--device /dev/ttyUSB0 is required to bind the serial port on the docker host device
--network hassio is the default docker bridge network for Home Assistant

nworbnhoj committed:
Update installing.rst

 --device /dev/ttyUSB0 is required to bind the serial port on the docker host device
--network hassio is the default docker bridge network for Home Assistant

nworbnhoj committed:
Update installing.rst

micolous commented:
It sounds like that the IP `172.30.33.0` isn't guaranteed to be consistent; eg: I might be using this subnet somewhere else in my network, which _generally_ causes Docker to avoid it.  I've used [RFC 5737 reserved IPs](https://tools.ietf.org/html/rfc5737) where something is "unknown".

As you probably guessed, I haven't looked at running these containers on `hassio` as yet - the C-Bus+HA environment I've looked after has HA running in a Docker container on a non-`hassio` Linux distribution, and has other non-HA-related containers running in it.

I'm not sure what to do about this; maybe the answer is to provide images that play nicely with `hassio`, but that'll take more effort. 😆

nworbnhoj commented:
I am new to HA, Docker, MQTT and cmqttd - and yes I am perhaps coming up against some of the challenges that you allude to in running cmqttd containers on hassio. Thank you very much for the work you have dome thus far in implementing a VERY welcome open-source cbus alternative.

I wonder if it might be possible to write a script to query the Docker DNS in order to identify any mqtt brokers on the hassio network, and then to configure cmqttd accordingly before launching? Do you think that this might be an approach worth pursuing?

nworbnhoj committed:
Improved docker run options in installing.rst

-d to run container in background
-it to enable re-attachment
--name cbus as a suitable container name
--restart=always to survive reboot
-b core-mosquitto to cleanly subscribe to mqtt broker based on Hostname

micolous commented:
I've incorporated your suggested change as 1d25bbb.  Thanks!
